### DISCUSSION TITLE
Will Craft support magic links or login codes to allow users to login using their email address

### DISCUSSION THREAD

**Discussion by zizther** [craftcms/cms#15390] [Ideas]
Will the Craft 5 support magic link or login codes on the CMS and frontend allowing users to access the site without the requirement for a password.

Examples of sites doing this is medium.com and poe.com

There is a plugin support magic links already, but support and features isn't as good as being a first party supported feature.

**Comment by brandonkelly** [comment]
We don’t have any current plans for it. Changed the discussion category to Ideas.
